### ISSUE TITLE
As a Consumer, I can perform a search by school size

### ISSUE THREAD

**Issue by ErieMeyer** [18F/college-choice#279]
I want to be able to search schools by Undergraduate Student Body size, with the options

+ Small (< 2,000)
+ Medium (2,000–15,000), and
+ Large (> 15,000)

**Comment by shawnbot** [comment]
Blocked by #362.

**Comment by shawnbot** [comment]
An important implementation note, cc @ultrasaurus and @yozlet:

There's an edge case here for when somebody selects "small" *and* "large" sizes, which requires us being able to ask for rows with `size < 2000` *or* `size > 15000`. I don't think that use case is captured in #362 or 18F/open-data-maker#52, because my assumption has been that boolean operators would all be *and*'ed. This might not be the type of thing that we need to resolve for the beta, but I'm going to suggest some ideas for how to address it in that repo.

**Comment by shawnbot** [comment]
This will be ready for acceptance when #393 merges.

**Comment by LisaGee** [comment]
This query results in one school that appears to fall outside of the requested range with only 1,989 undergraduate students.

https://federalist.18f.gov/preview/18F/college-choice/staging/search/?state=RI&&degree=2..3&size=2000..15000&sort=2011.earnings.6_yrs_after_entry.mean_earnings.lowest_tercile

**Comment by ultrasaurus** [comment]
https://github.com/18F/college-choice/pull/487 addressed the issue Lisa found.  Thanks @shawnbot for your helpful trail of FIXMEs.  

Opened separate issue for searching for small and large (without medium): https://github.com/18F/college-choice/issues/488
